cocos/physics-2d/framework/physics-system.ts

```diff
@@ -182,7 +183,7 @@ export class PhysicsSystem2D extends Eventify(System) {
      * @zh
      * 获取预定义的物理分组。
      */
-    public static get PhysicsGroup () {
+    public static get PhysicsGroup (): typeof PhysicsGroup {
```

Comment:
What's the type of PhysicsGroup?

Reply:
this is a trick on TS, if a variable is declare as `typeof PhysicsGroup`, that means this variable could be assigned with `PhysicsGroup` just like `const a: typeof PhysicsGroup  = PhysicsGroup`

we don't even need to know what's the type of `typeof PhysicsGroup`

This trick can help us implement the internal enumeration class of the class in TS